### PR TITLE
update docker-inspect man page

### PR DESCRIPTION
**Pull request by hqhq**
- sort inspect out
- update output fields
- format output
- add doc about go template
- other minor fix

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: docs/man/docker-inspect.1.md [hunk 1]
```diff
@@ -30,69 +30,109 @@ each result.
 
 To get information on a container use it's ID or instance name:
```
Comment (moxiegirl) [inline docs/man/docker-inspect.1.md]: `it's` > `its` can you change that while  you are in there.  Also, why was it necessary to change this example output so extensively?

File: docs/man/docker-inspect.1.md [hunk 2]
```diff
@@ -145,92 +175,92 @@ To get the IP address of a container use:
...
+        "Created": "2015-04-07T05:34:39.079489206Z",
+        "DockerVersion": "1.5.0-dev",
+        "Id": "fc1203419df26ca82cad1dd04c709cb1b8a8a947bd5bcbdfbef8241a76f031db",
+        "Os": "linux",
+        "Parent": "c2b774c744afc5bea603b5e6c5218539e506649326de3ea0135182f299d0519a",
+        "Size": 0,
+        "VirtualSize": 613136466
     }]
 
 # HISTORY
```
Comment (moxiegirl) [inline docs/man/docker-inspect.1.md]: Add a quick note to the history too please.

File: docs/man/docker-inspect.1.md (hunk 1 above)
Comment (hqhq) [inline docs/man/docker-inspect.1.md]: Nothing special, I was reading this man page to figure out how I can write this `-f` and didn't get the answer. I think we should add something, then I found some format issues and the outdated inspect out, so just change them at one time.
The example output don't need to be updated in time, but it's too old now, I think it's time to refresh. :)

**Comment by hqhq** [comment]
@moxiegirl Updated, thanks.

**Comment by moxiegirl** [comment]
Super! Thank you. LGTM

@thaJeztah @jamtur01 @fredlf @SvenDowideit

**Comment by jamtur01** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM

**Pull request merged**